src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import DropdownMenu from '../../../../components/dropdown-menu';
+import Checkmark16 from '../../../../icon/fill/Checkmark16';
+import IconEllipsis from '../../../../icons/general/IconEllipsis';
+import Pencil16 from '../../../../icon/line/Pencil16';
+import PlainButton from '../../../../components/plain-button';
+import { ButtonType } from '../../../../components/button';
+import Trash16 from '../../../../icon/line/Trash16';
+import X16 from '../../../../icon/fill/X16';
+import { Menu, MenuItem } from '../../../../components/menu';
+import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
+import { COMMENT_STATUS_OPEN, COMMENT_STATUS_RESOLVED } from '../../../../constants';
+import { bdlGray50 } from '../../../../styles/variables';
+import type { FeedItemStatus } from '../../../../common/types/feed';
+import messages from './messages';
+import './AnnotationActivityMenu.scss';
+
+type AnnotationActivityMenuProps = {
```

Comment:
Please export this as `interface`:
```suggestion
export interface AnnotationActivityMenuProps = {
```